packages/dev/parcel-transformer-docs/DocsTransformer.js

```diff
@@ -116,11 +117,15 @@ module.exports = new Transformer({
       }
 
       if (path.isTSTypeReference()) {
+        visitedTypes.add(path.node);
```

Comment:
without this hack, this ends up in a circular type reference which crashes the docs.

with the hack, the prop type table appears as empty.

so, we should figure out what to do here.